src/pages/Vote.tsx

```diff
@@ -81,6 +82,9 @@ const Vote = () => {
         onSubmit: () => {},
     });
 
+    const hasValues = formik.values.delegateAddress && formik.values.fee;
+    const hasSufficientFunds = BigNumber.make(wallet.balance()) > BigNumber.make(formik.values.fee);
```

Comment:
```suggestion
    const hasSufficientFunds = BigNumber.make(wallet.balance()).isGreaterThan(BigNumber.make(formik.values.fee));
```
otherwise it will do a string comparison, meaning `3 > 200`